.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: cpp
 
-# against hwloc error (travis-ci/travis-ci#10019)
-sudo: required
+dist: bionic
```

Comment:
Tests pass for the default dist (xenial), but not for the bionic dist. I'm still not sure why. Seeing a failure in `Cajita_BovWriter_test_OPENMP_1` with the message `*** stack smashing detected ***: <unknown> terminated`

I thought this may be due to the difference in gcc used by these dists? However, tests all pass on the LANL Darwin machine with gcc 7.4.0 (used by bionic) and gcc 5.4.0 (used by xenial)... ??? @junghans might have more expertise to share on this issue.

It seems that just using xenial here would get around test failures, although I would feel better knowing more about the cause of this error.

Reply:
I should probably run valgrind on that test and see what it tells me

Reply:
Not sure, we can revert the dist bump, but let's first try to use the default gcc instead of gcc-5.4.0

Reply:
Nope still stack smashing....

Reply:
Otherwise let's revert the bionic bump and fix that later, PR is already heavy enough.

Reply:
I just ran valgrind for this test on a different machine - everything came back clean. What are the odds it is the MPI implementation? This directly uses the MPI data type and MPI file I/O.